app/controllers/posts_controller.rb

```diff
@@ -2,11 +2,14 @@ class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
     @posts = @user.posts
+    @comments = Comment.all
+    @users = User.all
```

Comment:
- [ ] Kindly implement a solution for the N+1 problem. Currently, you're getting back all of the comments and users which isn't the solution. You can use [`includes`](https://apidock.com/rails/ActiveRecord/QueryMethods/includes) to fix the N+1 problem.

Reply:
Fixed.